docs/Customs Declare Exports AutoComplete.js

```diff
@@ -42,8 +43,6 @@ function dropDown() {
     var selectList = document.createElement("select");
     selectList.style.position = "absolute"
     selectList.style.top = "100px"
-    selectList.style.right = "5px"
-    selectList.style.zIndex = "999"
```

Comment:
This has been added by our designers. Are you sure that it's not necessary?

Reply:
it works fine, for me, but we will see :)